homeassistant/components/sensibo/climate.py

```diff
@@ -369,3 +375,8 @@ async def async_disable_pure_boost(self) -> None:
 
         await self.async_send_command("set_pure_boost", {"enabled": False})
         await self.coordinator.async_refresh()
+
+    async def async_reset_filter(self) -> None:
+        """Reset filter service."""
+        await self.async_send_command("reset_filter")
+        await self.coordinator.async_refresh()
```

Comment:
This looks like it could be a button entity ?

Reply:
> This looks like it could be a button entity ?

Remind me to ask you before next time 😛 
You are of course right 👍